src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+
+import * as FaIcons from 'react-icons/fa'
+import * as AiIcons from 'react-icons/ai'
+import PropTypes from 'prop-types'
+
+import SearchBar from '../SearchBar/SearchBar.component'
+import ToggleSwitch from '../ToggleSwitch'
+
+import Colors from '../../styles/Colors'
+import Button from '../Button'
+
+function Header(props) {
```

Comment:
**Optional**:

you can destructure the props so it's easier to tell which props are expected. In this case we're only passing down `func` so instead of 
```js
function Header(props)
```
you could have
```js
function Header({func})
```

Reply:
I totally agree, the code should be descriptive.

Reply:
https://github.com/G-Dyz/react-certification-2021/blob/c46af309f75c1e2d0259b18d4bcb503afd6847f2/src/components/Header/Header.component.jsx#L48